src/test/resources/vault/stores/pdbs/bags/9da0541a-d2c8-432e-8129-979a9830b427/metadata/files.xml

```diff
@@ -4,7 +4,7 @@
         <dcterms:type>http://schema.org/VideoObject</dcterms:type>
         <dcterms:format>video/mpeg</dcterms:format>
         <dcterms:title>hubble.mpg</dcterms:title>
-        <dcterms:accessRights>RESTRICTED_GROUP</dcterms:accessRights>
+        <accessibleToRights>RESTRICTED_GROUP</accessibleToRights>
```

Comment:
when you change this, please also add the specific scheme(s). See for examples `easy-split-multi-deposit` test data.

Reply:
@rvanheest : do you mean the XML namespace for `files.xsd`?

Reply:
yeah, that one. Couldn't think of the word. [Here's the link to the lines I meant.](https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/master/src/test/resources/allfields/output/input-ruimtereis01/bag/metadata/files.xml#L2-L3)